index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
Did you commit this by accident? The debug pages are in `debug/` we could probably add this functionality into the existing `debug/markers.html` page or if you don't want to you could leave it out for a future PR.